packages/main/src/plugin/updater.ts

```diff
@@ -62,6 +64,22 @@ export class Updater {
     this.#updateCheckResult = undefined;
   }
 
+  public async openReleaseNotes(version: string): Promise<void> {
+    if (version === 'current') {
+      version = app.getVersion();
+    } else if (version === 'latest') {
+      version = this.#nextVersion?.substring(1) ?? '';
+    }
+    const urlVersionFormat = version.split('.', 2).join('.');
+    let notesURL = `https://podman-desktop.io/blog/podman-desktop-release-${urlVersionFormat}`;
```

Comment:
I think here we should rely on the field `homepage` of the package.json file

(it is missing now but you can add it there: `https://www.podman-desktop.io`